.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "10.15"
+
+install:
+  - npm ci
+  - npm i -g @angular/cli
+
+branches:
+  only:
+    - master
+    - /^IO2019TEAM-.*$/
+
+script:
+  - npm run generate.declarations
+  - npm run build
+  - npm run lint
```

Comment:
@szbrnl Tutaj nie ma nic innego, co jest warte puszczenia?

Reply:
nie mamy żadnych testów, więc chyba nie

zastanawiam się jeszcze, czy build jest potrzebny, bo sprawdzany jest kod, a nie pliki wynikowe

Reply:
No to usuwam

Reply:
A build by się nie wywalił, jakby coś było nie tak?

Reply:
prawda, jednak się przyda xd

Reply:
@MrMarosis To przywracamy?

Reply:
No to przywracam :D